src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/Nav/BitNavDemo.razor

```diff
@@ -62,22 +62,6 @@
         </ExamplePreview>
     </ComponentExampleBox>
 
-    <ComponentExampleBox Title="Basic Nav, Top Of The Another Element" HTMLSourceCode="@example6HTMLCode" CSharpSourceCode="@example1CSharpCode" ExampleId="example6">
```

Comment:
you need to remove the `example6HTMLCode` from the code behind and rename example7 to example6 afterward.